executor/runtime/docker/capabilities.go

```diff
@@ -58,9 +59,12 @@ func setupAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.H
 	if c.TitusInfo.GetAllowNestedContainers() {
 		apparmorProfile = "docker-nested"
 		seccompProfile = "nested-container.json"
+		c.Env["TINI_UNSHARE"] = trueString
```

Comment:
We can rip out the nested containers code if you want?

Reply:
Are we never going to use it again?  I wasn't sure.

Reply:
We might, but for now, it's cruft that's not supported, and should never be set. Probably worth just getting rid of the code and throwing an error if we find it's set to true.